### DISCUSSION TITLE
Useful resources for the WG

### DISCUSSION THREAD

**Discussion by wd15** [marda-alliance/phase-field-schema#5] [General]
Placeholder for useful resources to inform WG.

 - [*The Semantic Data Dictionary – An Approach for Describing and Annotating Data*](https://direct.mit.edu/dint/article/2/4/443/94892/The-Semantic-Data-Dictionary-An-Approach-for).
 - [*How to Make a Data Dictionary*](https://help.osf.io/article/217-how-to-make-a-data-dictionary)
 - [*Codemeta*](https://codemeta.github.io/codemeta-generator/)
 - [*Existing PFHub schema*](https://github.com/usnistgov/pfhub-schema)
 - [*The role of metadata ...*](https://doi.org/10.1016/j.patter.2021.100322)
 - [*Packaging research artefacts with RO-Crate*](https://doi.org/10.3233/DS-210053)
 - [*Sharing interoperable workflow provenance*](https://doi.org/10.1093/gigascience/giz095)
 - [*Elementary Multiperspective Material Ontology*](https://github.com/emmo-repo/EMMO)
 - [*Ontology-based approach for the provision of simulation knowledge acquired by Data and Text Mining processes*](https://www.sciencedirect.com/science/article/pii/S1474034618304270)
 - [*A Framework for automated finite element analysis with an
ontology-based approach*](https://doi.org/10.1007/s12206-009-1005-0)
 - [*Best Practices of Ontology Development*](https://www.nist.gov/system/files/documents/2021/10/14/nist-ai-rfi-cubrc_inc_002.pdf)
 - [*Physics-based simulation ontology*](https://doi.org/10.1080/09544828.2019.1644301), [PDF](https://damassets.autodesk.net/content/dam/autodesk/research/publications-assets/pdf/physics-based-simulation-ontology.pdf)
 - [*Report on Workshop on Interoperability in Materials Modelling*](http://dx.doi.org/10.5281/zenodo.1240229)
 - [*Domain-specific metadata standardization in materials modelling*](http://dx.doi.org/10.5281/zenodo.4571052)
 - [*The Semantic Data Dictionary – An Approach for Describing and Annotating Data*](https://doi.org/10.1162/dint_a_00058)
 - [*Recording provenance of workflow runs with RO-Crate*](https://doi.org/10.48550/arXiv.2312.07852)

**Comment by tkphd** [comment]
## "Shared metadata for data-centric materials science"

[Ghiringhelli _et al._, _Scientific Data_ __10__ (2023) 626](https://doi.org/10.1038/s41597-023-02501-8) (PDF on [arXiv](https://arxiv.org/pdf/2205.14774.pdf))

![s41597-023-02501-8](https://github.com/marda-alliance/phase-field-schema/assets/2179347/171dd163-bfd8-4628-b80c-1ce82bc9fd38)

> The expansive production of data in materials science, their widespread sharing and repurposing requires educated support and stewardship. In order to ensure that this need helps rather than hinders scientific work, the implementation of the FAIR-data principles (Findable, Accessible, Interoperable, and Reusable) must not be too narrow. Besides, the wider materials-science community ought to agree on the strategies to tackle the challenges that are specific to its data, both from computations and experiments. In this paper, we present the result of the discussions held at the workshop on “Shared Metadata and Data Formats for Big-Data Driven Materials Science”. We start from an operative definition of metadata, and the features that  a FAIR-compliant metadata schema should have. We will mainly focus on computational materials-science data and propose a constructive approach for the FAIRification of the (meta)data related to ground-state and excited-states calculations, potential-energy sampling, and generalized workflows. Finally, challenges with the FAIRification of experimental (meta)data and materials-science ontologies are presented together with an outlook of how to meet them.

**Comment by tkphd** [comment]
## "Physics-based simulation ontology: an ontology to support modelling and reuse of data for physics-based simulation"

[Cheong & Butscher, _Journal of Engineering Design_ __30__ (2019) 655-687](https://doi.org/10.1080/09544828.2019.1644301) (PDF from [autodesk](https://damassets.autodesk.net/content/dam/autodesk/research/publications-assets/pdf/physics-based-simulation-ontology.pdf))

![physics-based-simulation-ontology](https://github.com/marda-alliance/phase-field-schema/assets/2179347/39acd491-80ae-4da3-a5e7-be001ab33963)

> The current work presents an ontology developed for physics-based simulation in engineering design, called Physics-based Simulation Ontology (PSO). The purpose of the ontology is to assist in modelling the physical phenomenon of interest in a veridical manner, while capturing the necessary and reusable information for physics-based simulation solvers. The development involved extending an existing upper ontology, Basic Formal Ontology (BFO), to define lower-level terms of PSO. PSO has two parts – PSO-Physics, which consists of terms and relations used to model physical phenomena based on the perspective of classical mechanics involving partial differential equations, and PSO-Sim, which consists of terms used to represent the information artefacts that are about the physical phenomena modelled with PSO-Physics. The former terms are used to model the physical phenomenon of interest independent of solver-specific interpretations, which can be reused across different solvers, while the latter terms are used to instantiate solver-specific input data. A case study involving two simulation solvers was conducted to demonstrate this capability of PSO. Discussion around the benefits and limitations of using BFO for the current work is also provided, which should be valuable for any future work that extends an existing upper ontology to develop ontologies for engineering applications.

**Comment by tkphd** [comment]
## "A framework for automated finite element analysis with an ontology-based approach"

[Sun, Ma & Chen, _Journal of Mechanical Science and Technology_ __23__ (2009) 3209](https://doi.org/10.1007/s12206-009-1005-0)

![s12206-009-1005-0](https://github.com/marda-alliance/phase-field-schema/assets/2179347/03de5ac7-5386-4f84-94b9-1662666ff306)

> To obtain accurate and reliable results of finite element analysis (FEA) requires a high level of expertise and the full-scale physical context information, which are bottlenecks restricting the application of achievements of FEA in industry. This paper proposes an ontology-based framework including a hierarchy transfer approach and a three-stage automated finite element analysis method to solve these problems. The hierarchy transfer approach is proposed to create different transfer formats according to the data, information, and knowledge, to carry out the integration at different levels. The knowledge found in design and FEA theories is presented by ontology in order to uniformly describe the physical phenomenon with the same semantic meanings. This involves the development of a shared design and FEA ontology, as well as, specific application ontologies in the Ontology Web Language (OWL). The three-stage automated finite element analysis method is applied to mark up artifact in problem definition, to reuse domain knowledge in problem formulation, and to enable the automation of the FEM analysis process in the solution routine with the application of AI techniques. The feasibility and effectiveness of the framework and concepts are empirically validated by a case study.
